docs/edex/new-grid.md

```diff
@@ -144,12 +93,16 @@ Notice that our grib2 file has a **Lat/lon Grid** projection.  Where:
 * **la1** is **40.799999**
 * **lo1** is **261**
 
-We will need these values for the next step. Note that **there is a tolerance of +/- 0.1 degrees** to keep in mind when defining your coverage (la1 and lo1) area.
+We will need these values for the next step. 
+
+!!! note "**There is a tolerance of +/- 0.1 degrees** to keep in mind when defining your coverage (la1 and lo1) area."
 
 ---
 
 ## Create Grid Projection File
     
+### Projection Types
+    
```

Comment:
For the projection types, maybe make a "Note":

You may not have information for every tag listed, for example it's not required for the latLonGridCoverage to have spacingUnit, la2, lo2

This was something Pete made a note about when referencing our documentation

Reply:
![image](https://user-images.githubusercontent.com/21114460/162518232-6efb46fc-d2a6-451a-b38b-2b542d37ab14.png)


Is this better?

Reply:
Yep, that looks good!